build/ical.js

```diff
@@ -1820,10 +1832,36 @@ ICAL.parse = (function() {
       }
       lcname = name.toLowerCase();
 
+      if (lcname in designSet.param && designSet.param[lcname].valueType) {
+        type = designSet.param[lcname].valueType;
+      } else {
+        type = DEFAULT_PARAM_TYPE;
+      }
+
+      if (lcname in designSet.param) {
+        multiValue = designSet.param[lcname].multiValue;
+      }
+
       var nextChar = line[pos + 1];
       if (nextChar === '"') {
         valuePos = pos + 2;
         pos = helpers.unescapedIndexOf(line, '"', valuePos);
+        if (multiValue && pos !== -1) {
+          var mvpos = pos;
+          var extendValue = true;
+          while (extendValue) {
+            var nextMDelim = helpers.unescapedIndexOf(line, multiValue, mvpos + 1);
+            var nextPDelim = helpers.unescapedIndexOf(line, PARAM_DELIMITER, mvpos + 1);
+            var nextVDelim = helpers.unescapedIndexOf(line, VALUE_DELIMITER, mvpos + 1);
+            var nextDQuote = helpers.unescapedIndexOf(line, '"', mvpos + 1);
+            if (nextDQuote > nextMDelim && (nextPDelim > nextDQuote || nextVDelim > nextDQuote)) {
```

Comment:
Does this also work if some delimiter are not found? Also, is there a chance that you can use the last found position as a starting point for the next indexOf call to save a few ops?